blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -105,9 +106,13 @@ export class RunDetailsPipeline extends Component {
     }
 
     componentWillUnmount() {
-        if (this.listener.sse) {
-            sseConnection.unsubscribe(this.listener.sse);
-            delete this.listener.sse;
+        if (this.listener.ssePipeline) {
+            sseConnection.unsubscribe(this.listener.ssePipeline);
+            delete this.listener.ssePipeline;
+        }
+        if (this.listener.sseJob) {
+            sseConnection.unsubscribe(this.listener.sseJob);
+            delete this.listener.sseJob;
```

Comment:
In general, I'd wrap cleanup code like this in try/finally blocks so as to ensure that if something fails in an early phase of the cleanup, that that doesn't cause an exception that causes the other phases of cleanup to be skipped. Prob not an issue here as I'm sure `sseConnection.unsubscribe` catches exceptions.